src/app/components/confirmBtcTransactionComponent/index.tsx

```diff
@@ -113,7 +113,7 @@ const ReviewTransactionText = styled.h1<ReviewTransactionTitleProps>((props) =>
 
 interface Props {
   currentFee: BigNumber;
-  feePerVByte: BigNumber;
+  feePerVByte: BigNumber; // is this the same as currentFeeRate?
```

Comment:
This is a strange setup. It looks like feePerVByte is the initial value received from the query params and is used to initialise the currentFeeRate in this component and in the fee editor.

Maybe we should refactor this? Out of scope for this PR bnut maybe change the comment into a `TODO`?